plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveConfig.java

```diff
@@ -176,6 +176,8 @@ public class HiveConfig
 
     private S3StorageClassFilter s3StorageClassFilter = S3StorageClassFilter.READ_ALL;
 
+    private int metadataParallelism = 8;
```

Comment:
I would drop the question from the commit message.
The question is

> Question: Should we consider setting the default value of
> "hive.metadata.parallelism" to 1 when using the "file" metastore?

I would not worry about file metastore. @raunaqmorarka WDYT?